WalletContext/pnpm-lock.yaml

```diff
@@ -0,0 +1,6460 @@
+lockfileVersion: 5.4
+
```

Comment:
we are using yarn everywhere, no?

Reply:
yes, sorry, will take this out in favor of yarn

Reply:
i hate to be super annoying, but ping here =p